CHANGELOG.md

```diff
@@ -61,6 +61,8 @@ release.
   ([#163](https://github.com/open-telemetry/semantic-conventions/pull/163))
 - Add .NET 8.0 metrics for HTTP client, ASP.NET Core, SignalR server and Kestrel.
   ([#283](https://github.com/open-telemetry/semantic-conventions/pull/283))
+- Add `aws.ecs.task.id` attribute, corrected description for `aws.ecs.task.arn`.
```

Comment:
There was a release in between, so please move the entry to the `Unreleased` section of the changelog.